installer/bundle/bundle_skel.sh

```diff
@@ -805,6 +829,23 @@ if [ "$installMode" = "I" -o "$installMode" = "U" ]; then
         echo "sed was not installed, installation cannot continue. Please install sed."
         cleanup_and_exit $INSTALL_SED
     fi
+    curl_installed
+    if [ $? -ne 0 ]; then
+        if [ -z "${forceFlag}" ]; then
+            echo "Error: curl is not installed, installation cannot continue."
+            echo "You can run this shell bundle with --force; in this case, we will install omsagent," >&2
+            echo "but omsconfig (DSC configuration) will not be available and will need to be re-installed." >&2
+            cleanup_and_exit $INSTALL_CURL
+        else
+            echo "curl is not installed,, please install curl and re-install omsconfig (DSC configuration) later."
+            echo "Installation will continue without installing omsconfig."
+        fi
+	fi
```

Comment:
Indentation seems inconsistent here; please check that all indents are 4 spaces.